EthOn_glossary.md

```diff
@@ -273,6 +276,9 @@ Relates a Post Transaction State to the following Post Transaction State.
 Relates a State to the following State. In EthOn the state transition system has no branches.
 ## O
 ## P
+### Prefunded Account
+[`ethon:PrefundedAccount`](http://ethon.consensys.net/PrefundedAccount)   
+A Prefunded Account is an Account whose initial balance is defined in the protocol of the Blockchain. In the initial Ethereum Blockchain those that participated in the crowd funding received Prefunded Accounts. However, Prefunded Accounts could be added with any hard fork.
```

Comment:
I think it may make more sense calling these differently, because they refer to the fact that they are not created through regular account creation means. Perhaps `IrregularlyCreatedAccount` is a more fitting term.

Such accounts can have a balance, can have code, can have storage. They can be defined as part of the *genesis file* or added via any hard fork.